### ISSUE TITLE
Report the best validation accuracy and loss (not the average of the last 5 epochs)

### ISSUE THREAD

**Issue by zhan2977** [keras-team/autokeras#640]
### Feature Description

Because autokeras uses early stopping. Will it be better to record the best validation loss and accuracy for each architecture? Currently, it records the average of the last 5 epochs.
